src/app/shared/model/feeder-podcast.model.ts

```diff
@@ -54,7 +55,12 @@ export class FeederPodcastModel extends BaseModel {
     }
     this.link = this.doc['link'] || '';
     this.newFeedUrl = this.doc['newFeedUrl'] || '';
-
+    if (this.series.has('prx:account')) {
+      this.series.follow('prx:account').subscribe(account => this.authorName = account['name']);
+    }
+    if (this.doc['author']) {
+      this.authorName = this.doc['author']['name'];
+    }
```

Comment:
Since that subscribe is technically asynchronous and continually subscribed, potentially could be overriding this.authorName. That's not actually what's happening because prx:account is embedded, so returns immediately, and doesn't update but I would prefer to see
```
if (this.doc['author'])
  this.authorName = this.doc['author']['name'];
else
  this.series.follow('prx:account').subscribe(account => this.authorName = account['name']);
```
I'm also thinking we've got to have that account info in the app already and shouldn't have to do that follow here. We can't inject the CmsService into this model like we can components. I think if this.series were a SeriesModel its parent should be the account. Hmm... we may need @cavis's advice on this.